calendar-bundle/src/Resources/contao/models/CalendarEventsModel.php

```diff
@@ -16,50 +16,50 @@
 /**
  * Reads and writes events
  *
- * @property string|integer      $id
- * @property string|integer      $pid
- * @property string|integer      $tstamp
- * @property string              $title
- * @property string              $alias
- * @property string|integer      $author
- * @property string|boolean      $addTime
- * @property string|integer|null $startTime
- * @property string|integer|null $endTime
- * @property string|integer|null $startDate
- * @property string|integer|null $endDate
- * @property string              $pageTitle
- * @property string              $robots
- * @property string|null         $description
- * @property string              $location
- * @property string              $address
- * @property string|null         $teaser
- * @property string|boolean      $addImage
- * @property string|boolean      $overwriteMeta
- * @property string|null         $singleSRC
- * @property string              $alt
- * @property string              $imageTitle
- * @property string|integer      $size
- * @property string              $imageUrl
- * @property string|boolean      $fullsize
- * @property string              $caption
- * @property string              $floating
- * @property string|boolean      $recurring
- * @property string              $repeatEach
- * @property string|integer      $repeatEnd
- * @property string|integer      $recurrences
- * @property string|boolean      $addEnclosure
- * @property string|array|null   $enclosure
- * @property string              $source
- * @property string|integer      $jumpTo
- * @property string|integer      $articleId
- * @property string              $url
- * @property string|boolean      $target
- * @property string              $cssClass
- * @property string|boolean      $noComments
- * @property string|boolean      $featured
- * @property string|boolean      $published
- * @property string|integer      $start
- * @property string|integer      $stop
+ * @property integer           $id
+ * @property integer           $pid
+ * @property integer           $tstamp
+ * @property string            $title
+ * @property string            $alias
+ * @property integer           $author
+ * @property string|boolean    $addTime
+ * @property integer|null      $startTime
+ * @property integer|null      $endTime
+ * @property integer|null      $startDate
+ * @property integer|null      $endDate
+ * @property string            $pageTitle
+ * @property string            $robots
+ * @property string|null       $description
+ * @property string            $location
+ * @property string            $address
+ * @property string|null       $teaser
+ * @property string|boolean    $addImage
+ * @property string|boolean    $overwriteMeta
+ * @property string|null       $singleSRC
+ * @property string            $alt
+ * @property string            $imageTitle
+ * @property integer           $size
```

Comment:
`tl_calendar_events.size` (and the other image size fields) is a `varchar` as well and is either a string containing a serialized array or a database ID - so this needs to stay `string|integer` as well.